src/main/java/org/craftercms/studio/api/v1/service/deployment/DeploymentService.java

```diff
@@ -73,7 +39,7 @@ List<ContentItemTO> getScheduledItems(String site, String sort, boolean ascendin
      *
      * @throws ServiceLayerException exception is case of en error
      */
-    void bulkGoLive(String site, String environment, String path, String comment) throws ServiceLayerException;
+    long bulkGoLive(String site, String environment, String path, String comment) throws ServiceLayerException, AuthenticationException;
```

Comment:
Rename to bulkPublish?

Reply:
@sumerjabri I would prefer to leave it like that so it matches the deprecated v1 API (`api/1/deployment/bulk-golive.json`) for the rest of its lifetime. This piece of code will be removed anyway when `bulk-golive` API is retired.